### PR TITLE
cdutil recipe

### PR DESCRIPTION
**Pull request by doutriaux1**

### CODE REVIEW

File: recipes/cdutil/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - numpy x.x
+    - udunits2
+    - genutil
+
+test:
+    command:
+        - export UVCDAT_ANONYMOUS_LOG=false && python run_tests.py -v2
+
+about:
+    home:  http://gitub.com/UV-CDAT/cdutil
```
Comment (synapticarbors) [inline recipes/cdutil/meta.yaml]: Mispelling: change `gitub` to `github`

File: recipes/cdutil/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    license: 'CCLRC'
+    summary: 'Climate Data Related Utitilites for the Community Data Analysis Tools'
+
+extra:
+  recipe-maintainers:
+    - dnadeau4
```
Comment (synapticarbors) [inline recipes/cdutil/meta.yaml]: @dnadeau4 - Are you ok being added as a co-maintainer of the `cdutil` recipe on conda-forge?

File: recipes/cdutil/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+build:
+  number: 0
+  skip: True  # [win or py3k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
```
Comment (synapticarbors) [inline recipes/cdutil/meta.yaml]: It's unclear to me if `numpy` or `genutil` are required for building this package from looking at the `setup.py`.

File: recipes/cdutil/build.sh [hunk 4]
```diff
@@ -0,0 +1 @@
+python setup.py install
```
Comment (synapticarbors) [inline recipes/cdutil/build.sh]: delete this file in favor of adding `script: python setup.py install` to the `build` section below the `skip` entry.

File: recipes/cdutil/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    - genutil
+  run:
+    - python
+    - cdms2
+    - numpy x.x
+    - udunits2
+    - genutil
+
+test:
```
Comment (synapticarbors) [inline recipes/cdutil/meta.yaml]: Perhaps add an import test here.

File: recipes/cdutil/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
+    - genutil
+  run:
+    - python
+    - cdms2
+    - numpy x.x
```
Comment (synapticarbors) [inline recipes/cdutil/meta.yaml]: Does this package require that a different version be built for different versions of numpy?

File: recipes/cdutil/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    - udunits2
+    - genutil
+
+test:
+    command:
+        - export UVCDAT_ANONYMOUS_LOG=false && python run_tests.py -v2
+
+about:
+    home:  http://gitub.com/UV-CDAT/cdutil
+    license: 'CCLRC'
```
Comment (synapticarbors) [inline recipes/cdutil/meta.yaml]: Please add `license_file: LICENSE`

File: recipes/cdutil/meta.yaml (hunk 2 above)
Comment (dnadeau4) [inline recipes/cdutil/meta.yaml]: Yes I am :+1:

File: recipes/cdutil/meta.yaml (hunk 5 above)
Comment (doutriaux1) [inline recipes/cdutil/meta.yaml]: tried it but in a virgin env we first ask user yes/no for some env to be set. I do not know how to pass an env variable to meta.yaml (google search suggest activate.d but i do not want to set the env variable to any default value for the distributed version)

**Pull request merged**